packages/common/utils/src/Numbers.ts

```diff
@@ -74,13 +74,7 @@ export function formatToKRW(
 }
 
 export function commaizeNumber(value: string | number) {
-  if (typeof value === 'string') {
-    return String(value).replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1,');
-  }
-
-  // maximumFractionDigits은 최대 소수점 자리수인데, default가 3입니다.
-  // 3자리보다 많은 수를 쓰는 경우가 있을 수도 있겠다고 생각해서 넉넉하게 10으로 잡았습니다.
-  return value.toLocaleString('en-us', { maximumFractionDigits: 10 });
+  return String(value).replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1,');
```

Comment:
This will make the result below:

```typescript
commaizeNumber('0.1234'); // '0.1,234'
```

How about split `value` by `.`?

Reply:
```typescript
const numStr = String(value);
const decimalPointIndex = numStr.indexOf('.');

if (decimalPointIndex > -1) {
  ...
} else {
  ...
}
```

Reply:
I fixed this code and check it works!
I checked other functions which using `commaizeNumber`. But, there are any edge cases can existed, so, I asked you for cross-check. Thanks for recommendation!

Reply:
How about adding some test cases to prove that `commaize` works correctly?

Reply:
```typescript
test('commaizeNumber', () => {
  expect(commaizeNumber(1234.1234)).toEqual('1,234.1234');
  expect(commaizeNumber(100)).toEqual('100');
  expect(commaizeNumber(13209802)).toEqual('13,209,802');
  expect(commaizeNumber('1234.1234')).toEqual('1,234.1234');
  expect(commaizeNumber('100')).toEqual('100');
  expect(commaizeNumber('13209802')).toEqual('13,209,802');
});
```

Reply:
I added some test cases and it works well!
Thank you!